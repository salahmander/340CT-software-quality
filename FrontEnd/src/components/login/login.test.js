import React from 'react';
import { shallow } from 'enzyme';
import ReactDOM from 'react-dom';

import Login from './login'

describe('item list ', () => {
    it('renders without crashing', () => {
        const div = document.createElement('div');
        ReactDOM.render(<Login />, div);
        ReactDOM.unmountComponentAtNode(div);
      });

    it('should render correctly in "debug" mode and match the snapshot', () => {
        const component = shallow(<Login debug />);
        expect(component).toMatchSnapshot();
      });
      
    });