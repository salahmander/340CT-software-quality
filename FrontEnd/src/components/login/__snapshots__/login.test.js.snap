// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`item list  should render correctly in "debug" mode and match the snapshot 1`] = `
<div
  className="logWrapper"
>
  <div
    className="form-wrapper"
  >
    <h1>
      Login
    </h1>
    <form
      noValidate={true}
      onSubmit={[Function]}
    >
      <div
        className="username"
      >
        <input
          name="username"
          noValidate={true}
          onChange={[Function]}
          placeholder="username"
          type="text"
        />
        <span />
      </div>
      <div
        className="password"
      >
        <input
          name="password"
          noValidate={true}
          onChange={[Function]}
          placeholder="Password"
          type="password"
        />
        <span />
        <span />
      </div>
      <div
        className="loginAccount"
      >
        <button
          className="logSumbit"
          type="submit"
        >
          Login
        </button>
        <button
          className="regButton"
          onClick={[Function]}
        >
          <small>
            Don't have an account?
          </small>
        </button>
      </div>
    </form>
  </div>
</div>
`;
